docs: update the error code (#321)
bytebase/bytebase.com: content/docs/en/reference/error-code/overview.md

---
title: Error Code
---

> Bytebase has different error codes for different namespaces.

Reviewer: ```suggestion
Bytebase employs a plugin architecture and is organized by modules. Each module has its own error code namespace.
```

> - [Error Code for Bytebase Core](/reference/error-code/core): The error code for bytebase.

Reviewer: ```suggestion
- [Error Code for Bytebase Core](/reference/error-code/core): The error code for Bytebase Core.
```

> - [Error Code for SQL Advisor](/reference/error-code/advisor): The error code for schema review system.

Reviewer: ```suggestion
- [Error Code for SQL Advisor](/reference/error-code/advisor): The error code for Bytebase SQL Advisor.
```
